web/src/pages/starter-kits/general.tsx

```diff
@@ -0,0 +1,158 @@
+import { NavBar } from "@/components/navbar/NavBar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { Card } from "@/components/starter-kits/Card";
+import { StepInfo } from "@/components/starter-kits/StepInfo";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { buildUserRoadmap } from "@/lib/roadmap/buildUserRoadmap";
+import { Roadmap, Task } from "@/lib/types/types";
+import analytics from "@/lib/utils/analytics";
+import { ProfileData, createEmptyProfileData } from "@businessnjgovnavigator/shared/index";
+import type { InferGetStaticPropsType } from "next";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
```

Comment:
[sand] `ReactElement` is a type so it should be imported using `import type`.

Reply:
new syntax unlocked 🧠